src/deploy/standalone/export.Dockerfile

```diff
@@ -0,0 +1,11 @@
+# This dockerfile exports just the single file executable from the built image.
```

Comment:
Do we need 2 Dockerfiles? 
if this is intended to be built using the makefile we can write it as an additional stage in `executable.Dockerfile` and in the make file use a target for building.

Reply:
I had this in the same dockerfile when I started, the problem is that when using docker build --output it seems to not save the cache because it only outputs the content to a directory, and I wanted to have a proper cache. So with an intermediate step that has a named image (`noobaa-core-executable`) the cache is saved and only the export step uses the --output flag.